### ISSUE TITLE
name of Cut object and name attribute can be different

### ISSUE THREAD

**Issue by mmarchegiani** [PocketCoffea/PocketCoffea#3]
The objects of type `Cut` defined in `parameters.cuts.baseline_cuts` can be assigned a name that is in general different from the object name (e.g. 'dilepton' is different from 'dilepton_presel').
To save the config file with `Configurator.save_config()` we need to access the name of the `Cut` object, therefore it would be nice to include an attribute similar to the attribute `__name__` for functions that would return the name of the object and not the attribute.

**Comment by valsdav** [comment]
```def load_cuts_and_categories(self):
        for cat, cuts in self.cfg["categories"].items():
            self.categories[cat] = []                
            for cut in cuts:
                # Check that the name of the cut is the same as the
                # label, to enforce a good bookeeping
                for obj_label, obj in locals().items():
                    if obj == cut:
                        if cut.name != obj_label:
                            print(f"PROBLEM: cut {cut} name is different than its label")
             
  ```
This code snippet is able to check if the label of a Cut is different from the `Cut.name` attribute. 

Anyway I'm not sure we would want to enforce that. Having the name in the Cut object is useful for bookeeping and having a different object label would not be the best. 

But imagine the case where the categories dictionary is created dyinamically. 
```
"categories" : {
"differential-1" : get_SR_categories() # some parametrized binning returning a [Cut,..]
}

In this case the object label of the cut is never used and we rely, correctly, on the internal of the Cuts for bookeping in the Configurator.  The obj label is just needed if the Cuts are imported from a library of them. 






```

**Comment by mmarchegiani** [comment]
Solved by [PR #16](https://github.com/PocketCoffea/PocketCoffea/pull/16)
